concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1814,14 +1969,17 @@ jobs:
             - -c
             - |
               VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-              CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.atc.basic_auth_password concourse-manifest/concourse-manifest.yml)
+              CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.basic_auth_password concourse-manifest/concourse-manifest.yml)
               export CONCOURSE_ATC_PASSWORD
               echo Looking for non running VMs
-              ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
+              echo 2 | ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
```

Comment:
Does using `<pipeline>/<resource>` work? Using a name would be more clear and reliable than using a number.